src/atopile/cli/configure.py

```diff
@@ -109,7 +109,7 @@ def install_kicad_plugin() -> None:
     """Install the kicad plugin."""
     # Find the path to kicad's plugin directory
     plugin_loader = f"""
-        plugin_path = "{Path(__file__).parent.parent}"
+        plugin_path = r"{Path(__file__).parent.parent}"
```

Comment:
We could even take it one further and just cast is to a string!

```suggestion
        plugin_path = str(Path(__file__).parent.parent)
```

Thank you Nick!!!

Reply:
![image](https://github.com/atopile/atopile/assets/51719418/c562cfab-fb7f-451c-8543-c6bdeeb8980b)

Won't this still treat the `\n` as an escape character?

Reply:
Nah, it won't, but by pulling it to test the idea I realised what was wrong with it; we're dumping this whole contents to a file anyway!

You're on the money with this fix already

Thank you Nick!